bee-ternary/src/bigint/endianness.rs

```diff
@@ -0,0 +1,30 @@
+// Copyright 2020 IOTA Stiftung
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+// the License. You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+// an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and limitations under the License.
+
+//! Endianness markers for big integers.
+
+use crate::bigint::sealed::Sealed;
+
+trait Endianness: Sealed {}
```

Comment:
this trait is not public, so why seal it? Sealing is useful for public traits, that users should not implement themselves on their own types, right? But this trait is in no danger of being implement as it's private.

Reply:
Yeah I think you're right. It was already there so I didn't question it too much. Maybe while cleaning it I removed a `pub` and didn't remove the seal. But the thing is that the trait is actually useless, it's never being used. So maybe  the seal was initially added by thinking it would be used in a public API and never was. I think I can just remove the trait...

Reply:
yea, I think that makes sense.

Reply:
yea, I think that makes sense.

Reply:
why did it repost my previous comment? 😂

Reply:
spam 😠